go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -912,6 +923,44 @@ func (qre *QueryExecutor) execShowMigrationLogs() (*sqltypes.Result, error) {
 	return nil, vterrors.New(vtrpcpb.Code_INTERNAL, "Expecting SHOW VITESS_MIGRATION plan")
 }
 
+func (qre *QueryExecutor) execShowThrottledApps() (*sqltypes.Result, error) {
+	if !qre.tsv.lagThrottler.IsEnabled() {
+		return nil, throttle.ErrThrottlerNotEnabled
+	}
+	if !qre.tsv.lagThrottler.IsOpen() {
+		return nil, throttle.ErrThrottlerNotEnabled
+	}
+	if _, ok := qre.plan.FullStmt.(*sqlparser.ShowThrottledApps); ok {
```

Comment:
nit: I believe it's idiomatic in golang to check and exit early.  I suggest flipping the above to:

```go
if _, ok := qre.plan.FullStmt.(*sqlparser.ShowThrottledApps); !ok {
    return nil, vterrors.New(vtrpcpb.Code_INTERNAL, "Expecting SHOW VITESS_THROTTLED_APPS plan")
}
```
```

Reply:
fixed, now doing an early return